js/functions.js

```diff
@@ -27,3 +27,37 @@ const getNumber = (string) => {
 };
 
 getNumber('skndslf');
+
+// Проверка, укладывается ли встреча в рабочий день - 1 вариант
+
+const getDurationInMinutes = (string) => {
+	const timeMassive = string.split(':');
+	return Number(timeMassive[0]) * 60 + Number(timeMassive[1]);
+};
+
+const isMatched = (dayBegin, dayEnd, meetBegin, meetDuration) => {
+	const meetEnd = getDurationInMinutes(meetBegin) + meetDuration;
+	return ((getDurationInMinutes(dayBegin) <= getDurationInMinutes(meetBegin))) && (meetEnd <= getDurationInMinutes(dayEnd));
```

Comment:
Не считаем дважды одно и то же
```suggestion
	const meetStart = getDurationInMinutes(meetBegin)
	const meetEnd = meetStart + meetDuration;
	return ((getDurationInMinutes(dayBegin) <= meetStart)) && (meetEnd <= getDurationInMinutes(dayEnd));
```